modules/swagger-codegen/src/main/resources/csharp/ApiClient.mustache

```diff
@@ -262,12 +270,14 @@ namespace {{packageName}}.Client
         /// <param name="name">Parameter name.</param>
         /// <param name="stream">Input stream.</param>
         /// <returns>FileParameter.</returns>
-        public FileParameter ParameterToFile(string name, Stream stream)
-        {
-            if (stream is FileStream)
+        public FileParameter ParameterToFile(string name, Stream stream) {
+            log.Trace("start");
+            if (stream is FileStream) {
+                log.Debug("FileStream");
```

Comment:
The word `FileStream` here wouldn't be useful in logs unless it says what method it is called from, and more about what the log message means.

Because log messages exist in asynchronously executed methods and can be called asynchronously by a client, you could end up with a log output like:

```
…
start
end
start
end
end
FileStream
…
```